adobe/spectrum-web-components
# fix(action-menu): allow tray to display full width

## Description
Ensure visual delivery of Tray when opening Action Menu in mobile

## Related issue(s)
- fixes #3822 

## How has this been tested?
    1. Go [here](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-menu--default)
    2. Switch to "mobile" view in dev tools
    3. Open the Action Group
    4. See erroneous delivery width of the Tray
    1. Go [here](https://picker-tray--spectrum-web-components.netlify.app/storybook/index.html?path=/story/action-menu--default)
    2. Switch to "mobile" view in dev tools
    3. Open the Action Group
    4. See the correct delivery of the Tray
    5. Do this

## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
